### ISSUE TITLE
Age expression

### ISSUE THREAD

**Issue by FranKusmiruk** [SkriptLang/Skript#3068]
### Description

A way to set the age of an ageable item type/block/entity.

Proposed syntax:
```vb
age[s] of %item types/blocks/entities%
%item types/blocks/entities%'[s] age[s]
```
Must be settable when possible.

### Workarounds
Although only for crops:
```vb
function changeStage(crop: itemtype, action: string = "add", by: number = 1) :: item type:
  set {_string} to "%{_crop}%"
  set {_stage::*} to {_string} parsed as "stage %number% %string%"
  set {_stage} to ({_stage::1} ? 1) + ({_by} if {_action} is "add", else {_by}*-1)
  
  if {_stage} is 1:
    return "freshly planted %{_stage::2}%" parsed as item type
  return "stage %{_stage}% %{_stage::2}%" parsed as item type

# Usage
set target block to changeStage(target block) # this will add 1
set target block to changeStage(target block, "rem") # this will remove 1
set target block to changeStage(target block, "rem", 3) # this will remove 3
```
*This is actually a reminder for myself but if anyone wants to attempt it, go ahead.*

**Comment by Runakai1** [comment]
Adding a general level expression for data values would be cool. Such as getting the fill level of a cauldron

**Comment by AyhamAl-Ali** [comment]
This issue is only partially PR Available #4594 but still missing `entities` support which I am working on already
